.github/workflows/dev.yml

```diff
@@ -0,0 +1,96 @@
+name: Dev
+concurrency: dev
+
+on:
+  push:
+    branches:
+      - master
+      - 'release/*'
+  pull_request:
+    branches:
+      - master
+      - 'release/*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Inject slug/short variables
+        uses: rlespinasse/github-slug-action@v3.x
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Login to Docker Hub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+      - name: Build and Push Temporary Tag
+        uses: docker/build-push-action@v2
+        with:
+          push: true
+          tags: squidex/squidex:tmp-dev-${{ env.GITHUB_SHA_SHORT }}
+          build-args: "SQUIDEX__VERSION=4.0.0-dev-${{ env.GITHUB_SHA_SHORT }}"
+          cache-from: type=registry,ref=squidex/squidex:cache-dev
+          cache-to: type=registry,ref=squidex/squidex:cache-dev
+
+  test:
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Inject slug/short variables
+        uses: rlespinasse/github-slug-action@v3.x
+      - name: Replace Image Name
+        uses: mikefarah/yq@v4.9.1
+        with:
+          cmd: yq e '.services.squidex.image = "squidex/squidex:tmp-dev-${{ env.GITHUB_SHA_SHORT }}"' -i backend/tests/docker-compose.yml
```

Comment:
Can we just simple increment version as we have it now. I use this approach in another project:

https://github.com/notifo-io/notifo/blob/main/.github/workflows/deploy.yml#L16

Reply:
Sure!

Reply:
Please use the trick with the increment, I don't want that dev starts from 1 again.

Reply:
Done